cargo-dist/tests/snapshots/cli_tests__manifest.snap

```diff
@@ -35,10 +35,10 @@ stdout:
           "description": "Install prebuilt binaries via powershell script"
         },
         {
-          "name": "cargo-dist-v1.0.0-FAKEVERSION-x86_64-pc-windows-msvc.zip",
```

Comment:
these snapshot diffs kill me (does --arifact all always do alphabetical?)

Reply:
I keep adding more sorting to make this stuff more stable but I think in this case it's using the order in the Cargo.toml (where previously it was using the order in the --target flags I was passing).